pkg/scheduler/apis/config/types.go

```diff
@@ -138,6 +138,9 @@ type KubeSchedulerProfile struct {
 // Enabled plugins are called in the order specified here, after default plugins. If they need to
 // be invoked before default plugins, default plugins must be disabled and re-enabled here in desired order.
 type Plugins struct {
+	// Enqueue is a list of plugins that should be invoked before adding pods to the scheduling queue.
```

Comment:
we need to add the new extension to https://github.com/kubernetes/kubernetes/blob/abcbaed0784baf5ed2382aae9705a8918f2daa18/pkg/scheduler/apis/config/types.go#L229

Reply:
done.